docs/logical_data_model.puml

```diff
@@ -236,6 +246,17 @@ class Files{
  * updatedAt : timestamp with time zone
 }
 
+class GoalCollaborators{
+ * id : integer : <generated>
+ * collaboratorTypeId : integer : REFERENCES "CollaboratorTypes".id
+ * goalId : integer : REFERENCES "Goals".id
+ * userId : integer : REFERENCES "Users".id
+ * createdAt : timestamp with time zone : now()
+ * updatedAt : timestamp with time zone : now()
+ deletedAt : timestamp with time zone
+ linkBack : jsonb
```

Comment:
I think we should have comments explaining the thinking behind linkbacks

Reply:
this is a generated file, so.... no comments here